src/utils/defaultLanguageStrings.ts

```diff
@@ -208,6 +208,7 @@ export default {
   LABEL_VERIFICATION_FAILED: 'Verificaton Failed',
   LINK_NEW_ACCOUNT: 'Create new account',
   LINK_RESET_PASSWORD: 'Forgot my password',
+  LOCALE: 'en-GB',
```

Comment:
This shouldn't be a string. It should be in the top-level "languages.json" file, e.g.:
```
  {
    "languageName": "Portugese",
    "description": "Portugese translation for Angola users",
    "code": "pt_an",
    "flag": "🇦🇴",
    "locale": "pt_PT"
    "enabled": true
  },
```

Fetch in the Language Provider and make it part of the global Language state.

Reply:
Yes, that's cool. I think we can actually use the existing `code` field, no?

Reply:
Oh, no I see we use that to differentiate 2 types of the same language too...

Reply:
> Yes, that's cool. I think we can actually use the existing `code` field, no?

Yeah, originally I thought the "code" would be equivalent to the locale, but we've kind of diverged from that.